usage/launcher/src/main/java/org/apache/brooklyn/launcher/BrooklynLauncher.java

```diff
@@ -710,32 +719,42 @@ private void initManagementContext() {
         }
     }
 
-    private boolean fileExists(String file) {
-        return new File(Os.tidyPath(file)).exists();
+    /**
+     * @return The canonical path of the argument.
+     */
+    private File resolveSymbolicLink(File f) {
+        File f2 = f;
+        try {
+            f2 = f.getCanonicalFile();
+            if (Files.isSymbolicLink(f.toPath())) {
+                LOG.debug("Resolved symbolic link: {} -> {}", f, f2);
+            }
+        } catch (IOException e) {
+            LOG.warn("Could not determine canonical name of global properties file", e);
```

Comment:
Perhaps change log message to not say `global properties file`, as this method is more generic (even if currently only called in that one case)?

How about `Could not determine canonical path of file "+f, e`